system/datetime-plugin/plugin/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "datetime-plugin",
-  "description": "Plugin to get current date and time",
```

Comment:
I think this was better description

Reply:
fixed. I updated the version from demos, not the one in integrations, and that's why this was changed.